tests/framework/helpers/ConsoleTest.php

```
@@ -440,6 +440,169 @@ public function testSelect(): void
+        ConsoleProgressStub::startProgress(0, 100, 'Drop: ', 80);
+        $this->truncateStreams();
+        ConsoleProgressStub::endProgress(true, false);
+        $this->assertStringNotContainsString('Drop:', $this->readOutput());
+        $this->truncateStreams();
+    }
+
+    public function testProgressBarETA(): void
+    {
+        ConsoleProgressStub::$screenSize = [80, 24];
+
+        ConsoleProgressStub::startProgress(0, 100, '', 80);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'yiiunit\framework\helpers\ConsoleProgressStub' in method 'testProgressBarETA'. (undefined)

(StaticAccess)